### DISCUSSION TITLE
Telegram Bot Discussion

### DISCUSSION THREAD

**Discussion by RioProjectX** [community/community#18654] [Other Feature Feedback, Questions, & Ideas]
hey come on disussion

**Comment by narmeranyo** [comment]
hi

**Comment by Gameye98** [comment]
Could someone share to me a code for telegram bot file uploader?

**Comment by Gameye98** [comment]
I do need it rn

**Comment by sudo-nautilus** [comment]
Well, um I guess this is not the suitable place to discuss about that, this should be a discussion about GitHub and things related to that, I believe.
 
Btw you can query about telegram bots in GitHub, and can find discussions in the right places in the discussions tab, https://github.com/search?q=telegram+bot

Also, @Gameye98 I think this can help you:
https://github.com/search?q=telegram+file+uploader
